lib/tasks/email_domain_actions.rake

```diff
@@ -0,0 +1,144 @@
+# -*- encoding : utf-8 -*-
+namespace :email_domain_actions do
+
+  desc "Lists top 20 most used email domains"
+  task :top20 => :environment do
+    # the limit of 20 is fairly arbitrary but I imagine this being more
+    # annoying than useful otherwise
+
+    from = ENV["START_DATE"]
+
+    sql = if from
```

Comment:
It would be nice to see stuff like this in a lib class with methods that just return data, so that they can be tested (particularly valuable if we're about to ban a domain's worth of users :)). Don't care about testing the output formatting though.